packages/api/src/routes/medical/patient.ts

```diff
@@ -203,6 +203,46 @@ const resourceSchema = z.enum(resourceTypeForConsolidation).array();
 
 /** ---------------------------------------------------------------------------
  * GET /patient/:id/consolidated
+ * @deprecated use /patient/:id/consolidated/query instead
+ */
+router.get(
+  "/:id/consolidated",
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getCxIdOrFail(req);
+    const patientId = getFrom("params").orFail("id", req);
+    const patient = await getPatientOrFail({ cxId, id: patientId });
+    return res.json({
+      status: patient.data.consolidatedQuery?.status ?? "not-started",
+      message:
+        "WARNING - this endpoint has been deprecated; please use GET /patient/:id/consolidated/query instead",
+    });
+  })
+);
+/** ---------------------------------------------------------------------------
+ * GET /patient/:id/consolidated/query
+ *
+ * Triggers a patient's consolidated data query. Results are sent through Webhook.
+ * If the query is already in progress, just return the current status.
```

Comment:
does this need to be updated to include a `@deprecated` flag?

Reply:
I was thinking to have this as a way for cxs to get a query status. Like, "The webhook is taking a long time, let me double check the query is still in progress or not".

Thoughts?

Reply:
Ah I get the intention here now, I just saw the `WARNING` message in the above func and just assumed the deprecation was needed here as well - the approach makes sense